README.md

```diff
@@ -31,10 +31,10 @@ Use latest release for particular version and/or channel:
 
 ```yaml
 steps:
-- uses: actions/checkout@v3
+- uses: actions/checkout@v4
 - uses: subosito/flutter-action@v2
   with:
-    flutter-version: '1.22.x'
```

Comment:
We need to keep this value, since this is for `dev` channel.

Reply:
Thanks for pointing that out.
I have restored `1.22.x`. [f53c1c7](https://github.com/subosito/flutter-action/pull/266/commits/f53c1c7363b8e2055d197a4ebba743e716de92b6)

Reply:
Thanks @koji-1009